lib/haml_lint/tree/tag_node.rb

```diff
@@ -186,7 +186,7 @@ def remove_inner_whitespace?
     #
     # @return [true,false]
     def remove_outer_whitespace?
-      @value[:nuke_outer_whitespace]
+      !!@value[:nuke_outer_whitespace]
```

Comment:
This is the most controversial thing that I changed. In Haml 5, this value apparently defaults to `nil` instead of a Boolean value. This was the simplest thing that worked, so I went with it. If we want to approach it from a different angle, I'm perfectly happy to switch it up.